Frontend/src/Modals/LoginAndRegisterModal.jsx

```diff
@@ -1,8 +1,122 @@
+import { useState, useRef } from 'react';
 import BaseModal from '../Base/BaseModal';
+import LoginAndRegister from '../LoginAndRegisterComponents/LoginAndRegister';
+import BaseButton from '../Base/BaseButton';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import { modalsActions } from '../Redux/modalsSlice';
 
 const LoginAndRegisterModal = (props) => {
+  const [title, setTitle] = useState('Sign In');
+  const [isLogging, setIsLogging] = useState(true);
+  const [isValidEmail, setIsValidEmail] = useState(null);
+  const [isValidPassword, setIsValidPassword] = useState(null);
+  const [isValidConfPassword, setIsValidConfPassword] = useState(null);
+  const emailRef = useRef(null);
+  const passwordRef = useRef(null);
+  const confPasswordRef = useRef(null);
+  const dispatch = useDispatch();
+
```

Comment:
W tej linii można zrobić skróconego if'a żeby mniej linii było ale to już bardziej moja preferencja.

Reply:
Jakiego ifa?

Reply:
Linia 22. W złym miejscu komentarz wpiąłem.

Reply:
Oki jak wrócę do domu to zobaczę